packages/vuetify/src/components/VDialog/VDialog.sass

```diff
@@ -18,6 +18,16 @@
   > *
     width: 100%
 
+  > .v-card
+    > .v-card__title
+      font-size: 1.25rem
```

Comment:
These are the headings.h6 font size settings. Please update it to use them:

```sass
.v-card__title
  font-size: $headings.h6.size
  font-weight: $headings.h6.weight
```

etc

Reply:
Done!